utils/check_ptc.py

```diff
@@ -0,0 +1,89 @@
+""" Tool for sanity checking rrgraph CHAN PTC's.
+"""
+import lxml.etree as ET
+import argparse
+
+
+def check_ptc(xml):
+    """ Checks ptc values used on CHANX and CHANY rr graph nodes are valid.
+
+    CHAN ptc numbers are an index per x/y coordinate and channel type (CHANX or
+    CHANY).  ptc's at a particular coordinate/type must start at 0, and fill
+    to the max value.
+
+    """
+    chan_ptcs = {}
```

Comment:
Maybe `chan_ptcs = defaultdict(list)` (unclear if it's better than just the if test....)